### ISSUE TITLE
oembed cache

### ISSUE THREAD

**Issue by anbininf12** [RRZE-Webteam/fau-oembed#14]
wäre es möglich im oembed-plugin einen Schalter einzubauen, um den oembed cache zu löschen?

**Comment by xwolfde** [comment]
Leider ist das nicht so trivial machbar, zumal es gerade an der Stelle viel Bewegung gibt.  ggf. eher Teil der globalen Cache-Behandlung @rvdforst ?

**Comment by rvdforst** [comment]
Genau. Es geht hier um die globale Variable $GLOBALS['wp_embed']. Die Standard-Cache-Zeit beträgt 24 Stunden und kann mit dem Filter 'oembed_ttl' (WP_Embed-Klasse) angepasst werden. Aber das Setzen der Cache-Zeit allein erzeugt den Cache nicht neu. Dies liegt an ($this->usecache || $cached_recently) (Z. 236). Die Regeneration der Cache-Zeit ist nicht trivial. Zusätzlicher Code ist erforderlich.
